app/src/main/java/com/example/ohthmhyh/HabitEvent.java

```diff
@@ -169,6 +188,7 @@ public void setBitmapPic(Bitmap pic) {
         UploadTask uploadTask = imgref.putBytes(picBytes);
     }
 
+    // TODO: remove this when refactoring how habit events are made.
     /**
      * Get the flag for this habitEvent
      * @return The flag for this habitEvent
```

Comment:
Wait, what is this flag used for again?

Reply:
I honestly don't know. It's something Stuart was using, but it shouldn't be needed. I think it was used to tell what habit event was clicked on in the list?